### PR TITLE
The Elevate field bundle cannot be used in Sandboxes that have Enhanced Domains enabled

### PR DESCRIPTION
**Pull request by balsamhindi**
[W-11652305
](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001515XIYAY/view)
Updated the lightning origin URL format to include sandbox/canonical URLs.
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/pages/GE_TokenizeCard.page [hunk 1]
```diff
@@ -208,15 +208,17 @@
...
         function buildLightningOrigin() {
-            const hostname = window.location.hostname;
-            const arr = hostname.split(".");
-            let domain = arr[0].replace('--npsp', '');
-            domain = domain.replace('--c', '');
             const siteBaseURL = '{!$Site.BaseUrl}';
+            let orgUrl = '{!OrgUrl}';
+            const sforceBaseUrl = '{!SFBaseUrl}';
+
+            if(sforceBaseUrl.includes("vf.force.com")){
```
Comment (balsamhindi) [inline force-app/main/default/pages/GE_TokenizeCard.page:215]: I'm not super confident that this logic will consistently get us what we want but it allows for dynamic URL creation

Comment (voduyemi) [reply force-app/main/default/pages/GE_TokenizeCard.page:215]: I think we should write a comment clearly stating why we went this route here.

**Review by voduyemi — APPROVED** [review]

File: force-app/main/default/pages/GE_TokenizeCard.page (hunk 1 above)
Comment (balsamhindi) [reply force-app/main/default/pages/GE_TokenizeCard.page:215]: That is a very good point, I can do that!

**Comment by balsamhindi** [comment]
@voduyemi Just as a heads up I went ahead and added some origin URLs as a precaution, nothing functionally changing, just as a "just in case" thing.

File: force-app/main/default/lwc/psElevateTokenHandler/psElevateTokenHandler.js [hunk 2]
```diff
@@ -50,25 +50,27 @@ class psElevateTokenHandler {
     getVisualForceOriginURLs(domainInfo, namespace) {
         const url = `https://${domainInfo.orgDomain}--${namespace}.visualforce.com`;
         const alternateUrl = `https://${domainInfo.orgDomain}--${namespace}.${domainInfo.podName}.visual.force.com`;
+        const lightningUrl =  `https://${domainInfo.orgDomain}--${namespace}.lightning.force.com`;
         const productionEnhancedUrl = `https://${domainInfo.orgDomain}--${namespace}.vf.force.com`;
         const productionEnhancedUrlLogin = `https://${domainInfo.orgDomain}--${namespace}.my.salesforce.com`;
         const sandboxEnhancedUrl =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.vf.force.com`;
-        const sandboxEnhancedUrlLogin =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.my.salesforce.com`;
-        const sandboxEnhancedUrlExperience =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.my.site.com`;
+        const sandboxEnhancedLightning =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.lightning.force.com`;
```
Comment (balsamhindi) [inline force-app/main/default/lwc/psElevateTokenHandler/psElevateTokenHandler.js:57]: @voduyemi / @daniel-fuller These are the changes that are added as a precaution

**Pull request merged**